Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/TimeAndDateExtensions.kt

```diff
@@ -112,6 +112,10 @@ object TimeAndDateExtensions {
 
     fun Instant.toLocalDateUserTz(): LocalDate = this.toUserTimeZone().toLocalDate()
 
+    fun LocalDate.isDifferentDay(other: LocalDate): Boolean {
+        return this.dayOfYear != other.dayOfYear || this.year != other.year
```

Comment:
Equals does not work?

Reply:
I thought this wouldn't work, but you are right, it does work. Removed the extension function again.